### ISSUE TITLE
`pulumi import --help` doesn't explain how to use it

### ISSUE THREAD

**Issue by dixler** [pulumi/pulumi#10288]
### What happened?

the help text for `pulumi import --help` explains the details of how it works, but doesn't explain how to use it   or what the `usage` text means.

```
Import resources into an existing stack.

Resources that are not managed by Pulumi can be imported into a Pulumi stack
using this command. A definition for each resource will be printed to stdout
in the language used by the project associated with the stack; these definitions
should be added to the Pulumi program. The resources are protected from deletion
by default.

Should you want to import your resource(s) without protection, you can pass
`--protect=false` as an argument to the command. This will leave all resources unprotected.

A single resource may be specified in the command line arguments or a set of
resources may be specified by a JSON file. This file must contain an object
of the following form:

    {
        "nameTable": {
            "provider-or-parent-name-0": "provider-or-parent-urn-0",
            ...
            "provider-or-parent-name-n": "provider-or-parent-urn-n",
        },
        "resources": [
            {
                "type": "type-token",
                "name": "name",
                "id": "resource-id",
                "parent": "optional-parent-name",
                "provider": "optional-provider-name",
                "version": "optional-provider-version",
                "properties": ["optional-property-names"],
            },
            ...
            {
                ...
            }
        ]
    }

The name table maps language names to parent and provider URNs. These names are
used in the generated definitions, and should match the corresponding declarations
in the source program. This table is required if any parents or providers are
specified by the resources to import.

The resources list contains the set of resources to import. Each resource is
specified as a triple of its type, name, and ID. The format of the ID is specific
to the resource type. Each resource may specify the name of a parent or provider;
these names must correspond to entries in the name table. If a resource does not
specify a provider, it will be imported using the default provider for its type. A
resource that does specify a provider may specify the version of the provider
that will be used for its import.
Each resource may specify which input properties to import with;
If a resource does not specify any properties the default behaviour is to
import using all required properties.

Usage:
  pulumi import [type] [name] [id] [flags]
```

### Steps to reproduce

`pulumi import --help`

### Expected Behavior

Help to be much shorter and to have example usages

### Actual Behavior

Is mainly prose and details around implementation.

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by dixler** [comment]
@cnunciato I saw that @t0yv0 linked you on 
- https://github.com/pulumi/pulumi/issues/9677

and thought this might be under the same umbrella.

**Comment by key88sf** [comment]
```
pulumi import aws:ecr:Repository ecr-my-repo-name 123456789
```

fails with ` error: Preview failed: unrecognized resource type (Read): aws:ecr:Repository`.   Any idea what is wrong with this? I think it follows the proper format, no?

**Comment by key88sf** [comment]
Finally figured out the syntax which should be:
```
pulumi import aws:ecr/repository:Repository ecr-my-repo-name 123456789
```

Really would like it if the docs were clearer about how to use this!

**Comment by cphoover** [comment]
I agree... the docs are really lacking here.

I think that tab completion or a resource search function would be very helpful as well for the CLI.

**Comment by lunaris** [comment]
I think this has been addressed in #10318 -- the help text for `import` now includes some examples, for instance.
